alacritty/src/logging.rs

```diff
@@ -122,13 +123,29 @@ impl log::Log for Logger {
 
         // Write to stdout.
         if let Ok(mut stdout) = self.stdout.lock() {
-            let _ = stdout.write_all(msg.as_ref());
+            let _ = stdout.write_all(msg);
         }
     }
 
     fn flush(&self) {}
 }
 
+fn append_log_message<'a>(msg: &'a mut String, record: &log::Record<'_>) -> &'a [u8] {
```

Comment:
Why isn't the time and format moved into this function? Wouldn't it be easier to just return a String here?